tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -110,7 +153,7 @@ param(
     }
 
     Write-Host "serviceAccountIssuer = $serviceAccountIssuer"
-    Restart-ApiServer -serviceAccountIssuer $serviceAccountIssuer
+    Restart-ApiServer -serviceAccountIssuer $serviceAccountIssuer -useK8s:$useK8s
 
     & $helmPath repo add azure-workload-identity https://azure.github.io/azure-workload-identity/charts
```

Comment:
just checking, the lines from 158-160 are temporary, right?

Reply:
yes. wil be removed with final bits from Arc team